mininet/node.py

```diff
@@ -1466,7 +1466,8 @@ def __init__( self, name, **kwargs ):
     def isAvailable( cls ):
         return (which( 'ovs-controller' ) or
                 which( 'test-controller' ) or
-                which( 'ovs-testcontroller' ))
+                which( 'ovs-testcontroller' ) or
+                which( 'ovs-ofctl' ))
```

Comment:
This is not correct - `ovs-ofctl` is not a controller.

Reply:
Now that I looked at it more carefully, I realised that my change does not make any sense.
By the way, what should not work without a proper ovs-testcontroller? I am using mininet so emulate a network on a continuous integration server based on Fedora with this incorrect hack, and it seems to work! At least the nodes are created and they communicate to each other. I am just curious.